apache/incubator-zeppelin
# [ZEPPELIN-760] Companion object bug fix for Flink Interpretter

### What is this PR for?
This implements the solution presented for the Spark interpreter in PR-780 for the Flink, Scalding, ~~and Ignite~~ interpreters.

### What type of PR is it?
Improvement

### Todos
* [x] - Fix line splitting 

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-658
* https://issues.apache.org/jira/browse/ZEPPELIN-747

### How should this be tested?
Add a new paragraph with each interpreter and define a class with its companion object.
See bug ZEPPELIN-658 for more details

### Screenshots (if appropriate)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No
